djangobench/benchmarks/default_middleware/urls.py

```diff
@@ -1,7 +1,10 @@
 try:
-    from django.conf.urls import url
+    from django.urls import re_path as url
```

Comment:
can we go the other way and import `url as re_path` for older versions?

Also maybe better to use version checking - `if django.VERSION >= (2, 0):` then import `re_path`

https://adamj.eu/tech/2020/01/07/feature-checking-versus-version-checking/

Reply:
Ok I switched everything from `url()` to `re_path()` and used version checking.